### PR TITLE
Filter by category

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Add a filter Functionality
-Filter by talks
-Filter by stock market
-Filter by crypto
-Filter by finance

### CODE REVIEW

File: src/utils/filterByCategory.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const filterByCategory = (products,type) => {
+    switch (type) {
+      case "talk":
```
Comment (harshitpaliwal95) [inline src/utils/filterByCategory.jsx:3]: use Upercase keys in types

**Review by harshitpaliwal95 — COMMENTED** [review]
every things looks good you can merge

**Review by zabihshaik — APPROVED** [review]
UI looks Good.
Folder structure is good.

File: src/components/Modal/Modal.css [hunk 2]
```diff
@@ -3,15 +3,15 @@
     width: 50%;
     position: absolute;
     top: 2%;
-     left: 30%;
+    left: 30%;
     border: 1px solid var( --primary-color);
     height: 45%;
     z-index: 99;
```
Comment (ravikumar1002) [inline src/components/Modal/Modal.css:9]: Avoid Random z-indices

File: src/components/navbar/navbar.jsx [hunk 3]
```diff
@@ -1,52 +1,63 @@
...
-</div>
-<div className="navbar__profile">
-  <Link to = "./login"> <span className="navbar__text">Login</span></Link> 
-</div>
-    </div>
+      <div className="navbar">
+        <Link to="/">
+          <div className="navbar__logo">
+            <div className="navbar__media">
+              <img src={logo} alt="" />
```
Comment (ravikumar1002) [inline src/components/navbar/navbar.jsx:16]: Avoid using empty string in alt attribute

File: src/components/navbar/navbar.jsx [hunk 4]
```diff
@@ -1,52 +1,63 @@
...
+              <span className="navbar__text">Watch later</span>
+            </Link>
+            <span className="navbar__badge_number navbar__text">
+              {watchLaterState.watchLaterItems.length}
+            </span>
+          </div>
+        </div>
+        <div className="navbar__profile">
+          <Link to="./login">
+            {" "}
```
Comment (ravikumar1002) [inline src/components/navbar/navbar.jsx:54]: remove this

**Review by ravikumar1002 — APPROVED** [review]
* well structured code
 * Liked folder structure.
 * type naming also nice.
 * overall its good, merge it.

**Pull request merged**